### ISSUE TITLE
NodeJS "--debug" parameter is deprecated

### ISSUE THREAD

**Issue by jmaister** [kiddkai/atom-node-debugger#234]
(node:2624) DeprecationWarning: node --debug is deprecated. Please use node --inspect instead.

**Comment by theantoniorocha** [comment]
Ran into the same problem today.

**Comment by oit63** [comment]
DeprecationWarning: `node --debug` and `node --debug-brk` are invalid. Please use `node --inspect` or `node --inspect-brk` instead.

**Comment by axe-z** [comment]
same here , never used this, but fixing the path , this is what I get, hard first run !

**Comment by Redsandro** [comment]
Can any of you get the debugger to work with Node 6.11? Even using `--inspect`, I can do breakpoints, but I cannot inspect any variables, as reported in issue https://github.com/kiddkai/atom-node-debugger/issues/237

**Comment by vildmedtog** [comment]
I'm using node v8.2.1 having the same prob.
maybe line 56 in debugger.coffee needs to change to "--inspect-brk"? perhaps it shouldn't be hard coded in a future release :-)

**Comment by Redsandro** [comment]
I think it isn't that simple. Otherwise I'd be able to inspect values when changing the command line manually.

**Comment by w0k3** [comment]
So the package is node v8 incompatible? Any news on this?

**Comment by nitrocode** [comment]
Yep I have the same issue. I tried changing line 56 but after reloading the app, it only loads up in chrome. I can't debug from within Atom using node 8 on osx.

**Comment by tb01923** [comment]
Hi adding in a +1 on the issue and looking for a solution.  I am new to Mac and Atom (new Job) so not sure how much I can add at this point other than I would like to be able to debug node 8 in atom.  Any information (such as why this is an issue) would be appreciated.  It is deeper than the command line option - right?

**Comment by skywinder** [comment]
+1  on Mac

**Comment by yatishdang** [comment]
Facing same issue !! Please share if anyone got solution .

**Comment by chucky86boy** [comment]
+1 on Mac using v8.9.3

**Comment by AbdelrahmanHafez** [comment]
+1 on Windows, using v8.9.3
Anyone found a solution?

**Comment by AbdelrahmanHafez** [comment]
So, one way to solve it, as @vildmedtog pointed out.

Go to your `.atom` directory, find `packages\node-debugger\lib\debugger.coffee` line 56, change `--debug` to `--inspect`.

~~That solved the issue for me.~~

**Edit:**
This makes the error go away, but I don't seem to get the debugger itself working, I assume this is because the node-inspector uses Chrome websocket debugger, not the old debugging method.
I suppose we'll need a new version of node-debugger after all.

**Edit 2018-06-17:**
Not sure if this type of comments is welcome here, but this is a personal opinion, VSCode gives a much better debugging experience, and a generally better editing experience, it probably would be a good idea to use it until this is fixed.

**Comment by relief-melone** [comment]
Same issue here. No fix still?

**Comment by nitrocode** [comment]
@relief-melone  Since I couldn't debug node 8 in atom, I use vscode and chrome dev tools now.

**Comment by BeardedDonut** [comment]
+1 on ubuntu 16.04 and Node v8

**Comment by FrancisTurner** [comment]
Any status on a fix? This issue renders the whole package worthless.

**Comment by dhowe** [comment]
Status??
